src/main/java/model/domain/Player.java

```diff
@@ -0,0 +1,42 @@
+package model.domain;
+
+import model.VO.Name;
+import model.VO.Position;
+import model.VO.Result;
+
+public class Player {
+    private final Name name;
+    private Result result;
+    private Position position;
+
+    public Player(final Name name, final Position position) {
+        this.name = name;
+        this.position = position;
+    }
+
+    public Name getName() {
+        return name;
+    }
+
+    public Result getResult() {
+        return result;
+    }
+
+    public void saveResult(final Result result) {
+        this.result = result;
+    }
+
+    public boolean isPlayerName(Name other) {
+        return other.isSame(this.name);
+    }
+
+    public boolean isPlayerPosition(Position otherPosition) {
```

Comment:
```suggestion
    public boolean isSamePosition(Position otherPosition) {
```

위와 같은 간결한 네이밍은 어떤가요 ?